daemon/logger/omslogs/omsclient/omsclient.go

```diff
@@ -0,0 +1,105 @@
+// OMS Data Collector package adapted from
+// https://github.com/Azure/oms-log-analytics-firehose-nozzle
```

Comment:
Not yet. We have started conversations with the Azure team to take ownership of one and would prefer to vendor it.

As to the `gofmt`, I'll do so when I get back from vacation. Having peeked at a few individual build failures, some seemed systemic or maybe the formatting error just wasn't as obvious in some environments as in others.

Thanks!